sdk/storage/Azure.Storage.DataMovement.Files.Shares/README.md

```diff
@@ -167,11 +165,13 @@ await directoryTransfer.WaitForCompletionAsync();
 
 ## Troubleshooting
 
-***TODO***
+See [Handling Failed Transfers](#handling-failed-transfers) and [Enabling Logging](https://learn.microsoft.com/en-us/dotnet/azure/sdk/logging) to assist with any troubleshooting.
 
 ## Next steps
 
-***TODO***
+Get started with our [Blob DataMovement samples][blob_samples].
+
+For more base Transfer Manager scenarios see [DataMovement samples][samples].
```

Comment:
Same two comments as with Blob samples but in this case the link exists but you deleted what it points to